### PR TITLE
Add proposed QC term spectra half-TIC

### PR DESCRIPTION
**Pull request by edeutsch**

### CODE REVIEW

**Review by mobiusklein — APPROVED** [review]

File: psi-ms.obo [hunk 1]
```diff
@@ -22663,6 +22663,18 @@ relationship: has_value_type xsd\:float ! The allowed value-type for this CV ter
 relationship: has_value_concept NCIT:C25330 ! Duration
 relationship: has_units UO:0000010 ! second
 
+id: MS:4000068
```
Comment (mobiusklein) [inline psi-ms.obo:22667]: Requires `[Term]` stanza opening syntax. I didn't see it before.

**Review by mobiusklein — APPROVED** [review]

File: psi-ms.obo (hunk 1 above)
Comment (edeutsch) [reply psi-ms.obo:22667]: great, thanks for fixing!

**Comment by mobiusklein** [comment]
@bittremieux if you can, please check that this conforms to the expected shape for mzQC

**Review by mwalzer — APPROVED** [review]
Thanks! This looks good.

**Review by bittremieux — APPROVED** [review]
Yes, looks good. Thanks!

**Pull request merged**